packages/resolve-scripts/src/server/render.js

```diff
@@ -55,14 +55,19 @@ export default (initialState, { req, res }) => {
       `${helmet.title.toString()}` +
       `${helmet.meta.toString()}` +
       `${helmet.link.toString()}` +
+      '<style>\n' +
+      'html, body, .app-container { width: 100%; height: 100%; padding: 0; border: none; ' +
```

Comment:
Why exactly these styles had been hard-coded in React template render? Are there some de-facto standard?